File: example/come_x_pay_y.js
function come_x_pay_y(pax, per_head, come=4, pay=3) {
    let amt = Math.floor(pax / come) * (pay * per_head) + (pax % come) * per_head;
    return amt;
}

console.log(come_x_pay_y(5, 100, 4, 3));
console.log(come_x_pay_y(5, 100));

File: example/come_x_pay_y.rs
fn come_x_pay_y(pax: i32, per_head: i32, come: i32, pay: i32) -> i32 {
	let amt = (pax / come) * (pay * per_head) + (pax % come) * per_head;
	return amt;
}

fn main() {
	println!("{}", come_x_pay_y(5, 100, 4, 3));
	// println!("{}", come_x_pay_y(5, 100));
}